tests/uptane/ats_director.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
As a general note to the all of these test `json` files, can we store the keys that were used to make them too that way we can use them in other tests?